Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -13276,6 +13225,41 @@ Ref<WebBackForwardList> WebPageProxy::protectedBackForwardList() const
     return m_backForwardList;
 }
 
+template<typename F>
+void WebPageProxy::sendToWebPage(std::optional<FrameIdentifier> frameID, F&& sendFunction)
+{
+    if (frameID) {
+        if (RefPtr frame = WebFrameProxy::webFrame(*frameID)) {
+            if (RefPtr remotePageProxy = frame->remotePageProxy()) {
+                sendFunction(remotePageProxy.get());
```

Comment:
Would it work to call sendFunction(*remotePageProxy) and sendFunction(*this) and have the lambda parameter be auto& instead of auto*?